theme/templates/resource-landing-page/title-section.html

```diff
@@ -327,7 +362,62 @@ <h2 id="resource-title">{{ title }}</h2>
         .title-builder-input {
             margin-top: 1em;
         }
+        .path-align {
+          word-wrap: break-word;
+          white-space: normal !important;
+        }
     </style>
+
+    <script>
+        let nonPreferredPathDiv = $("#non-preferred-paths");
+        nonPreferredPathDiv.hide();
+        if (CAN_CHANGE && !RESOURCE_PUBLISHED_OR_UNDER_REVIEW) {
+            let url = "/hsapi/_internal/" + SHORT_ID + "/non-preferred-paths/";
+
+            $.ajax({
+                type: "GET",
+                url: url,
+                dataType: 'json',
+                data: {},
+                async: true,
+                success: function (result) {
+                    $("#checking-for-paths").hide();
+                    result = JSON.stringify(result);
+                    let json_response = JSON.parse(result);
+                    if (json_response.status === "SUCCESS") {
+                        //$("#checking-for-paths").hide();
+                        if (json_response.non_preferred_paths.length) {
+                            nonPreferredPathDiv.show();
+                            let modalBody = nonPreferredPathDiv.find(".modal-body");
+                            let list = document.createElement("UL");
+                            for (let i = 0; i < json_response.non_preferred_paths.length; i++) {
+                                let listItem = document.createElement("LI");
+                                let code = document.createElement("CODE");
+                                let itemText = document.createTextNode(json_response.non_preferred_paths[i]);
+                                code.appendChild(itemText);
+                                listItem.appendChild(code);
+                                list.appendChild(listItem);
+                            }
+                            modalBody.append(list);
+                            modalBody.find("ul").children().each(function () {
```

Comment:
instead of finding your li and adding class after creating all of them, add the class up above in your for loop, where you generate the `li`

example:
```
let listItem = document.createElement("LI");
listItem.classList.add("path-align");
```

Reply:
Done.